### ISSUE TITLE
Spacy Time parsing

### ISSUE THREAD

**Issue by mike23333** [explosion/spaCy#2857]
The spacy time parsing is not getting all times correct. It misses many times. Any solutions? Also where is the training data for time parsing so I can add the examples it's not recognizing so it recognizes it.

**Comment by ines** [comment]
The process of recognising named entities is **statistical** – so the English model has no deeper knowledge of what a date or time entity is, or how a "person" is defined. It only *predicts* what label is most likely in this context and which span to apply it to, based on the data the model was trained on. In the case of the English models, spaCy uses the [OntoNotes 5](https://catalog.ldc.upenn.edu/LDC2013T19) corpus. It's usually free for research, but requires a license for commercial use.

It's important to keep in mind that the pre-trained models distributed with the library are baseline models that were tuned for the best possible compromise of speed, size, and accuracy and make it easy to get started building your own systems. You **almost always** want to adjust the model to your specific domain if extracting named entities is important to you. You can find more details on this in the [documentation on training and updating models](https://spacy.io/usage/training).

Depending on what you're trying to achieve, you might find that a rule-based approach actually works much better for you than a statistical model. Machine learning is great, but only really useful if you need your system to be able to *generalise* based on similar examples. If you have a more or less fixed set of terms or patterns that you want to extract from a text, the [rule-based `Matcher`](https://spacy.io/usage/linguistic-features#section-rule-based-matching) can be very powerful.
